src/retreival_chain.py

```diff
@@ -83,24 +87,44 @@ def create_retrieval_chain(
     # Get OpenAIEmbeddings (or HuggingFaceEmbeddings model if specified)
     embedding_callable = get_embedding(hf_model, device)
 
+    csv_file_mapping = {
+        "complexes": "complexes.csv",
+        "reactions": "reactions.csv",
+        "summations": "summations.csv",
+        "ewas": "ewas.csv",
+    }
+
     # Adjusted type for retriever_list
     retriever_list: list[RetrieverLike] = []
     for subdirectory in list_chroma_subdirectories(embeddings_directory):
+        # set up BM25 retriever
+        csv_file_name = csv_file_mapping[subdirectory]
+        reactome_csvs_dir: Path = EmbeddingEnvironment.get_dir("reactome") / "csv_files"
+        loader = CSVLoader(file_path=str(reactome_csvs_dir) + "/" + csv_file_name)
```

Comment:
I would remove `csv_file_mapping` above and use `Path` way of composing the file path like this:
```py
        # set up BM25 retriever
        csv_file_name = f"{subdirectory}.csv"
        reactome_csvs_dir: Path = embeddings_directory / "csv_files"
        loader = CSVLoader(file_path=reactome_csvs_dir / csv_file_name)
```